substrate/frame/revive/src/wasm/runtime.rs

```diff
@@ -237,8 +237,8 @@ parameter_types! {
 	const XcmExecutionFailed: ReturnErrorCode = ReturnErrorCode::XcmExecutionFailed;
 }
 
-impl From<ExecReturnValue> for ReturnErrorCode {
-	fn from(from: ExecReturnValue) -> Self {
+impl<'a> From<&'a ExecReturnValue> for ReturnErrorCode {
+	fn from(from: &'a ExecReturnValue) -> Self {
```

Comment:
```suggestion
impl From<&ExecReturnValue> for ReturnErrorCode {
	fn from(from: &ExecReturnValue) -> Self {
```
I think you can use elided lifetime here

Reply:
Yes, thanks!